_posts/2022-03-25-unnormalized-resnets.md

```diff
@@ -0,0 +1,725 @@
+---
+layout: post
+title: Normalization is dead, long live normalization!
+tags: [normalization, initialization, propagation, skip connections, residual networks]
+authors: Hoedt, Pieter-Jan; Klambauer, Günter; Hochreiter, Sepp
+---
+
+<style>
+    figcaption { color: gray; }
+</style>
+
+Since the advent of Batch Normalization (BN), almost every state-of-the-art (SOTA) method uses some form of normalization.
+After all, normalization generally speeds up learning and leads to models that generalize better than their unnormalized counterparts.
+This turns out to be especially useful when using some form of skip connections, which are prominent in Residual Networks (ResNets), for example.
+However, [Brock et al. (2021a)](#brock21characterizing) suggest that SOTA performance can also be achieved using **ResNets without normalization**!
```

Comment:
Could you swap these around to give a nice excerpt for the `blog/` endpoint?

i.e.:

```markdown

Since the advent of Batch Normalization (BN), almost every state-of-the-art (SOTA) method uses some form of normalization.
After all, normalization generally speeds up learning and leads to models that generalize better than their unnormalized counterparts.
This turns out to be especially useful when using some form of skip connections, which are prominent in Residual Networks (ResNets), for example.
However, [Brock et al. (2021a)](#brock21characterizing) suggest that SOTA performance can also be achieved using **ResNets without normalization**!

<style>
    figcaption { color: gray; }
</style>
```